libs/bulldozer/application/features/view-instruction/src/lib/view-instruction.component.ts

```diff
@@ -16,73 +16,95 @@ import {
-        [basicAccounts]="basicAccounts$ | ngrxPush"
-        [signerAccounts]="signerAccounts$ | ngrxPush"
-        [programAccounts]="programAccounts$ | ngrxPush"
-        (updateBasicAccount)="onUpdateBasicAccount($event)"
-        (deleteBasicAccount)="onDeleteBasicAccount($event)"
-        (updateSignerAccount)="onUpdateSignerAccount($event)"
-        (deleteSignerAccount)="onDeleteSignerAccount($event)"
-        (updateProgramAccount)="onUpdateProgramAccount($event)"
-        (deleteProgramAccount)="onDeleteProgramAccount($event)"
-      >
-      </bd-list-accounts>
-    </main>
+        </bd-instruction-menu>
+
+        <main>
+          <bd-list-arguments
+            class="block mb-4"
+            [connected]="connected$ | ngrxPush"
+            [arguments]="arguments$ | ngrxPush"
+            (updateArgument)="onUpdateArgument($event)"
+            (deleteArgument)="onDeleteArgument($event)"
+          ></bd-list-arguments>
+
+          <bd-list-accounts
+            class="block mb-16"
+            [connected]="connected$ | ngrxPush"
+            [accountsCount]="accountsCount$ | ngrxPush"
+            [basicAccounts]="basicAccounts$ | ngrxPush"
+            [signerAccounts]="signerAccounts$ | ngrxPush"
+            [programAccounts]="programAccounts$ | ngrxPush"
+            (updateBasicAccount)="onUpdateBasicAccount($event)"
+            (deleteBasicAccount)="onDeleteBasicAccount($event)"
+            (updateSignerAccount)="onUpdateSignerAccount($event)"
+            (deleteSignerAccount)="onDeleteSignerAccount($event)"
+            (updateProgramAccount)="onUpdateProgramAccount($event)"
+            (deleteProgramAccount)="onDeleteProgramAccount($event)"
+          >
+          </bd-list-accounts>
+        </main>
+      </div>
+      <div class="w-2/4">
+        <bd-code-editor
+          [customClass]="'bd-border-bottom custom-monaco-editor-splited'"
+          [template]="rustStaticCodeInstruction$ | ngrxPush"
+          [readOnly]="true"
+        ></bd-code-editor>
+        <bd-code-editor
+          [customClass]="'custom-monaco-editor-splited'"
+          [template]="rustDynamicCodeInstruction$ | ngrxPush"
+        ></bd-code-editor>
+      </div>
+    </div>
   `,
   styles: [],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class ViewInstructionComponent implements OnInit {
-  @HostBinding('class') class = 'block p-4';
+  @HostBinding('class') class = 'block';
   readonly connected$ = this._walletStore.connected$;
   readonly instruction$ = this._tabsStore.tab$;
   readonly arguments$ = this._instructionStore.arguments$;
   readonly basicAccounts$ = this._instructionStore.basicAccounts$;
   readonly signerAccounts$ = this._instructionStore.signerAccounts$;
   readonly programAccounts$ = this._instructionStore.programAccounts$;
   readonly accountsCount$ = this._instructionStore.accountsCount$;
+  readonly rustStaticCodeInstruction$ = this._instructionStore.rustCode$.pipe(
```

Comment:
you renamed the templates property to context but left the selector name as static, same with the handle/dynamic selector